docs/conversion/sakai_23_mysql_conversion.sql

```diff
@@ -0,0 +1,4 @@
+-- SAK-46974
+ALTER TABLE MFR_MESSAGE_T ADD COLUMN SCHEDULER bit(1) DEFAULT NULL;
```

Comment:
I would prefer to see this column not nullable, and with a default.

Reply:
Done